packages/usa-form/src/test/test-pattern/test-usa-form.twig

```diff
@@ -50,7 +50,7 @@
 </fieldset>
 
 
-<fieldset class="usa-fieldset margin-top-2">
+<fieldset class="usa-fieldset">
```

Comment:
@mahoneycm Can you also remove the margin on the checkbox component on this page?

Reply:
Resolved in [06d1cfa](https://github.com/uswds/uswds/pull/5344/commits/06d1cfa7c7501b67bcf5e25234f4c25040011c31)